src/service/resource.js

```diff
@@ -556,9 +556,10 @@ export class Resource {
    * @return {boolean}
    */
   isDisplayed() {
-    return (this.layoutBox_.height > 0 && this.layoutBox_.width > 0 &&
+    return ((this.element.getAttribute('height') == 'fluid' ||
```

Comment:
You've already precalculated layout, so let's use it here. Just expose it on CustomElement as `getLayout()`, then this will be:

```
return ((this.element.getLayout() == Layout.FLUID || ...)
```

Reply:
Done.